trade-finance-manager-ui/component-tests/utilisation-reports/_macros/report-reconciliation-table.component-test.js

```diff
@@ -35,6 +36,11 @@ describe(component, () => {
 
       wrapper.expectElement(`${rowSelector} th`).toHaveCount(1);
       wrapper.expectElement(`${rowSelector} th:contains("${summaryItem.bank.name}")`).toExist();
+      if (summaryItem.status === 'REPORT_NOT_RECEIVED') {
+        wrapper.expectLink(`${rowSelector} th > a`).toLinkTo(undefined, summaryItem.bank.name);
```

Comment:
Is this temporary behaviour or expected for this status? If we don't link anywhere for banks where report isn't received I think this shouldn't be an `a` element.

Reply:
So I tried to not make it an `a` element, but making it an `a` without an `href` attribute keeps the same text formatting except without the blue underlined colour

Reply:
Clearly a controversial decision! It turns out you can make sure the text has the same style by wrapping in a `<p class="govuk-body">` tag